README.md

```diff
@@ -216,15 +219,28 @@ Example:
 |:--:|
 | *Buddies collection* |
 
-## Example
+## Examples
 
-To launch the example, type the following commands in shell:
+To launch the examples, type the following commands in shell:
 
 ```bash
 npm install
 npm run dev-server
 ```
 
+You can find 2 examples:
+
+### The simplest use of the Widgets library
+
+In the file `examples/simplest_example_unlogged.html`, you can find the simplest usage of the library you can do. Without need to have token for app or for a user, there is no need to login user for this.
```

Comment:
Second sentence should read "Without the need to have a token for the app or for the user, this example doesn't need any login process."